scripts/setup/ubuntu-20.04/install_deps.sh

```diff
@@ -2,28 +2,34 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 
-set -eux
+set -eu
 
-# Install tools in Ubuntu 20.04 via `apt-get`
-sudo apt-get --yes update \
-  && sudo DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends --yes \
-  bison \
-  cmake \
-  ctags \
-  curl \
-  flex \
-  g++ \
-  gcc \
-  git \
-  gpg-agent \
-  libssl-dev \
-  lsb-release \
-  make \
-  ninja-build \
-  patch \
-  pkg-config \
-  python-is-python3 \
-  software-properties-common \
-  wget \
-  zlib1g \
+DEPS=(
```

Comment:
Let's keep these sorted alphabetically

Reply:
I think it's more helpful to group them by purpose first, then sort alphabetically, but I can see how one single dependency grouped separately might not make that obvious. I'll switch to alphabetically with a trailing comment for now.

(But for example, I can't find any flex/bison files in RMC. I tried removing those dependencies, and it seems like it rebuilt fine. I'm not sure why they're there, and it'd be nice if they were grouped under some header comment explaining why they were there. Possibly I just didn't rebuild everything?)